applications/rag/README.md

```diff
@@ -105,7 +105,11 @@ gcloud container clusters get-credentials ${CLUSTER_NAME:?} --location ${CLUSTER
 1. Verify Kuberay is setup: run `kubectl get pods -n ${NAMESPACE:?}`. There should be a Ray head (and Ray worker pod on GKE Standard only) in `Running` state (prefixed by `example-cluster-kuberay-head-` and `example-cluster-kuberay-worker-workergroup-`).
 
 2. Verify Jupyterhub service is setup:
```

Comment:
We also need to update the frontend instructions for IAP - see https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/applications/rag/README.md#launch-the-frontend-chat-interface